android/library/build.gradle

```diff
@@ -55,6 +56,7 @@ dependencies {
 	}
 
 	implementation libraries.appCompat
+	implementation libraries.design
```

Comment:
Is this library mandatory? I've removed it for avoiding having tons of extra methods

Reply:
Yes, for now, we are using snackbar feedbacks inside screens, but I think we should provide a way to the developer handle with the feedbacks instead of render snackbars.